robot_skills/src/robot_skills/arm/gripper.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, robot_name, tf_listener, gripper_name):
         """
         super(ParrallelGripper, self).__init__(robot_name=robot_name, tf_listener=tf_listener)
         self.gripper_name = gripper_name
-        offset = self.load_param('skills/arm/' + self.gripper_name + '/grasp_offset/')
+        offset = self.load_param('skills/' + self.gripper_name + '/grasp_offset/')
```

Comment:
I would say a gripper is part of an arm, so the parameter is also part in the parameter tree of the arm.

Reply:
It is not necessarily part of an arm. That is why we created a more composable robot interface